src/mds/topology/topology.cpp

```diff
@@ -892,6 +1038,7 @@ int TopologyImpl::Init(const TopologyOption &option) {
         return ret;
     }
 
+    WriteLockGuard wlockPoolset(poolsetMutex_);
```

Comment:
There is no new operation on poolset, why add a lock here？

Reply:
it is reference physicalPool

Reply:
> There is no new operation on poolset, why add a lock here？

it is reference physicalPool